p2p_wallet/Extensions/UIView+Extensions.swift

```diff
@@ -145,3 +149,15 @@ extension UIView {
         UIImageView(width: 6, height: 12, image: .nextArrow, tintColor: .h8b94a9.onDarkMode(.white))
     }
 }
+
+extension UIView {
+    
+    // Using a function since `var image` might conflict with an existing variable
+    // (like on `UIImageView`)
+    func asImage() -> UIImage {
+        let renderer = UIGraphicsImageRenderer(bounds: bounds)
```

Comment:
Now we use this method in more than 1 place, so please don't hesitate to remove the private extension in Settings.UsernameViewController.